proto/panacea/datapool/v2/tx.proto

```diff
@@ -31,7 +31,7 @@ service Msg {
 // MsgRegisterDataValidator defines the Msg/RegisterDataValidator request type.
 message MsgRegisterDataValidator {
   string address = 1;       // The address of data validator to be registered
-  string endpoint = 2;      // https://my-validator.org
+  DataValidator validator_detail = 2;      // https://my-validator.org
```

Comment:
I think it would be good to come here only as an endpoint as before.

Reply:
I just put only`DataValidator` field in `MsgRegisterDataValidator`. [551dd34](https://github.com/medibloc/panacea-core/pull/285/commits/551dd341b2af599174a28c493e717a8506f8c1d7)